subsys/dfu/dfu_target/Kconfig

```diff
@@ -37,6 +37,15 @@ if DFU_TARGET_MCUBOOT_SAVE_PROGRESS
 	comment "DFU_TARGET_MCUBOOT_SAVE_PROGRESS is deprecated, please use DFU_TARGET_STREAM_SAVE_PROGRESS instead"
 endif
 
+if DFU_TARGET_MCUBOOT
+config DFU_TARGET_MCUBOOT_DELIMITER
+	int "Delimiter for MCUBOOT S0 and S1 filenames"
+	default 32
+	help
+	  Use any ASCII character as delimiter.
```

Comment:
Could you provide more detail as to what this is used for? Anyone changing this also has to ensure that the URLs generated are correct.